x-pack/plugins/transform/server/routes/api/privileges.ts

```diff
@@ -38,40 +39,26 @@ export function registerPrivilegesRoute({ router, license }: RouteDependencies)
           },
         });
 
-      // Find missing cluster privileges and set overall app privileges
-      privilegesResult.missingPrivileges.cluster = extractMissingPrivileges(cluster);
-      privilegesResult.hasAllPrivileges = hasAllPrivileges;
-
       // Get all index privileges the user has
       const { indices } = await esClient.asCurrentUser.security.getUserPrivileges();
 
       // Check if they have all the required index privileges for at least one index
-      const oneIndexWithAllPrivileges = indices.find(({ privileges }: { privileges: string[] }) => {
-        if (privileges.includes('all')) {
-          return true;
-        }
-
-        const indexHasAllPrivileges = APP_INDEX_PRIVILEGES.every((privilege) =>
-          privileges.includes(privilege)
-        );
+      const hasOneIndexWithAllPrivileges =
```

Comment:
line 32 needs a type fix, GitHub won't let me add a comment.
```ts
const esClient: IScopedClusterClient = (await ctx.core).elasticsearch.client;
```
By specifying that `esClient` is a `IScopedClusterClient`, the other types are then correct.